emscripten.py

```diff
@@ -1866,10 +1865,9 @@ def build_wasm(temp_files, infile, outfile, settings, DEBUG):
 
 
 def build_wasm_lld(temp_files, infile, outfile, settings, DEBUG):
-  assert shared.Settings.BINARYEN_ROOT, 'need BINARYEN_ROOT config set so we can use Binaryen tools on the backend output'
-  wasm_emscripten_finalize = os.path.join(shared.Settings.BINARYEN_ROOT, 'bin', 'wasm-emscripten-finalize')
-  wasm_as = os.path.join(shared.Settings.BINARYEN_ROOT, 'bin', 'wasm-as')
-  wasm_dis = os.path.join(shared.Settings.BINARYEN_ROOT, 'bin', 'wasm-dis')
+  wasm_emscripten_finalize = os.path.join(shared.BINARYEN_ROOT, 'bin', 'wasm-emscripten-finalize')
```

Comment:
maybe an idea for a future refactoring, but perhaps this area could call `get_binaryen_bin()` in `tools/shared.py` which returns the bin dir, and we could make that function not fetch the port if binaryen is set in the config file.

seems nicer than to read `shared.BINARYEN_ROOT` directly. however, again, we can leave this to later work.